docs/docs-content/security-bulletins/security-advisories/security-advisories.md

```diff
@@ -11,6 +11,81 @@ tags: ["security", "cve", "advisories"]
 Security advisories supplement <VersionedLink text="security bulletins" url="/security-bulletins/reports/" />, providing
 additional details regarding vulnerabilities and offering remediation steps.
 
+## Security Advisory 005 - `runc` Container Vulnerabilities Affecting Kubernetes Cluster Runtimes
+
+- **Release Date**: November 5, 2025
+- **Last Update**: November 6, 2025
+- **Severity**: High
+- **Applicable Deployments**:
+  - Workload Clusters:
+    - cloud-managed clusters: EKS, GKE, and AKS
+    - Palette-managed clusters: PXK, RKE2, and K3s
```

Comment:
Is "Palette-managed clusters" the correct terminology to use here?

If it affects clusters using these Kubernetes distributions, perhaps that needs to be clearer.

Maybe:

- Kubernetes distributions/packs: PXK, RKE2, and K3s